README.md

```diff
@@ -39,8 +42,9 @@ Project page is located @ [https://vmware.github.io/docker-volume-vsphere/](http
 
 ## Supported Platform
 
-ESXi: 6.0 and above
-Docker: 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)
+**ESXi:** 6.0 and above<br />
+**Docker (Linux):** 1.12 and higher (Recommended 1.13/17.03 and above to use managed plugin)<br />
+**Docker (Windows):** 17.03 and above (Windows containers mode only)
```

Comment:
is there any dependency on docker 17.03? As per https://blog.docker.com/2016/09/build-your-first-docker-windows-server-container/ docker 1.12 should be fine too.